ext/random/php_random.h

```diff
@@ -218,11 +217,15 @@ typedef struct _php_random_status_state_user {
 	zend_function *generate_method;
 } php_random_status_state_user;
 
+typedef struct _php_random_result {
+	const uint64_t result;
```

Comment:
I've initially had it reversed, but this order minimizes the differences in the generated assembly, because `.result` stays in `rax` (with `.size` being in `rdx`, instead of vice versa).